checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
+that is shipped in the distribution and is intended for users of the
+Checker Framework.
+
+When adding a .jar file to this directory, update the list in README and also
```

Comment:
I am not aware of such instructions, but I don't think this is the right place to add them since the annotated JDK doesn't live in this directory.